Eatery/Onboarding/Controllers/OnboardingLoginViewController.swift

```diff
@@ -223,6 +246,30 @@ class OnboardingLoginViewController: OnboardingViewController {
         }
     }
 
+    @objc func keyboardWillShow(notification: NSNotification) {
+        guard let userInfo = notification.userInfo else { return }
+
+        guard let keyboardSize = userInfo[UIKeyboardFrameEndUserInfoKey] as? NSValue else { return }
+
+        let keyboardFrame = keyboardSize.cgRectValue
+
+        if self.view.frame.origin.y == 0 {
+            self.view.frame.origin.y -= keyboardFrame.height * 2 / 3
```

Comment:
This is a bit hacky; Its strange for a view controller to adjust its view's position inside its superview.

I think a way to avoid doing this would be to add keyboard logic into the `OnboardingViewController` and have it adjust the entire stackview upwards when it detects the keyboard.